src/libs/ajax/Metrics.ts

```diff
@@ -23,6 +23,7 @@ export const Metrics = (signal?: AbortSignal) => {
       event,
       properties: {
         ...details,
+        // TODO: Sup with this logic yo?
```

Comment:
It's probably because if the user is registered, the auth token is sent to Bard on L37 and we don't need to send `distinct_id`. Bard supports events from anonymous users via setting `distinct_id`, which can later be reconciled with the `/api/identify` endpoint. See Bard [README](https://github.com/DataBiosphere/bard) and [API docs](https://terra-bard-dev.appspot.com/docs/).